app/api/access_codes.py

```diff
@@ -96,6 +97,9 @@ class AccessCodeList(ResourceList):
     """
     List and create AccessCodes
     """
+    def before_post(self, args, kwargs, data):
```

Comment:
There is one more thing in marshmallow decorators, `post_load`. Can you please check if we can use that instead of calling befoe_post and before_patch?